apps/velo/save_file.py

```diff
@@ -3,32 +3,31 @@
 import sys
 
 
-def save_file(text, remote_path, site_user, velo_username, password, filename):
+def save_file(text, filename, site_user, velo_username, password):
     try:
         velo_api = VeloAPI.Velo()
         if not velo_api.isJVMStarted():
             velo_api.start_jvm()
         rs = velo_api.init_velo(velo_username, password)
 
-        local_path = os.getcwd() + 'userdata/acmetest/' + filename
+        local_path = os.getcwd() + '/userdata/' + velo_username + '/'
         try:
-            f = open(local_path, 'w')
+            f = open(local_path + filename, 'w')
             f.write(text)
             f.close()
         except:
-            print 'I/O failure when writing file'
+            print 'I/O failure when writing file to django server'
             raise
 
-        if velo_api.upload_file(remote_file_path, local_path):
+        if velo_api.upload_file(local_path, filename) >= 0:
+            print 'File saved'
             return 0
-            # c = content.splitlines(True)
-            # for line in c:
-            #     print line
         else:
+            print 'Error saving file', local_path, filename
             return -1
     except:
         raise
 
 
 save_file(sys.argv[1], sys.argv[2], sys.argv[3],
-          sys.argv[4], sys.argv[5], sys.argv[6])
+          sys.argv[4], sys.argv[5])
```

Comment:
Let's wrap all of these calls with a sys.exit so the return values actually get caught